src/sts.rs

```diff
@@ -0,0 +1,278 @@
+//! The AWS STS API.
+//!
+//! If you're using the service, you're probably looking for [StsClient](struct.StsClient.html).
+#![allow(unused_imports)]
```

Comment:
I removed the imports in question and all STS tests pass.  However they aren't as thorough as actually using the service.  Can you make those changes in your fork and verify it works with your STS project?